src/types/index.ts

```diff
@@ -186,25 +186,26 @@ export interface MarketSnapshot {
   updatedAt: string;
 }
 
-export type TransactionEvent =
-  | 'Mint'
-  | 'Transfer'
-  | 'Borrow'
-  | 'RepayBorrow'
-  | 'Redeem'
-  | 'Approval'
-  | 'LiquidateBorrow'
-  | 'ReservesAdded'
-  | 'ReservesReduced'
-  | 'MintVAI'
-  | 'Withdraw'
-  | 'RepayVAI'
-  | 'Deposit'
-  | 'VoteCast'
-  | 'ProposalCreated'
-  | 'ProposalQueued'
-  | 'ProposalExecuted'
-  | 'ProposalCanceled';
+export enum TransactionEvent {
+  Mint = 'Mint',
+  Transfer = 'Transfer',
+  Borrow = 'Borrow',
+  RepayBorrow = 'RepayBorrow',
+  Redeem = 'Redeem',
+  Approval = 'Approval',
+  LiquidateBorrow = 'LiquidateBorrow',
+  ReservesAdded = 'ReservesAdded',
+  ReservesReduced = 'ReservesReduced',
+  MintVAI = 'MintVAI',
+  Withdraw = 'Withdraw',
+  RepayVAI = 'RepayVAI',
+  Deposit = 'Deposit',
+  VoteCast = 'VoteCast',
+  ProposalCreated = 'ProposalCreated',
+  ProposalQueued = 'ProposalQueued',
+  ProposalExecuted = 'ProposalExecuted',
+  ProposalCanceled = 'ProposalCanceled',
+}
```

Comment:
Just curious: is there an advantage in using an enum rather than a list of strings?

For enums, I think we should follow the pattern you followed with the errors and uppercase + snake case each item.

Reply:
I went with an enum because places where the values were referenced  as plain strings it kept throwing a type error that string wasn't equal to a value in the union although they were the same